.env.example

```
@@ -11,5 +11,16 @@ DB_NAME={database}
 # Environment
 NODE_ENV=development
 
+# Cloudflare R2 (media storage) — required for `svelte-kit sync` to generate
+# correct $env/static/private and $env/static/public types, even as empty
+# placeholders (see src/lib/db/services/R2Service.ts, ContentService.ts).
+CLOUDFLARE_ACCOUNT_ID=
+R2_ACCESS_KEY_ID=
+R2_SECRET_ACCESS_KEY=
+PUBLIC_R2_IMAGES_BUCKET=
```

dotenv-linter (4.0.0)

[warning] [UnorderedKey] The PUBLIC_R2_IMAGES_BUCKET key should go before the R2_ACCESS_KEY_ID key

(UnorderedKey)